ironfish-cli/src/commands/repl.ts

```diff
@@ -54,13 +55,17 @@ export default class Repl extends IronfishCommand {
     this.log(`\n  Use the RPC node/getStatus`)
     this.log(`  > (await client.status()).content`)
     this.log('')
+    this.log(`\n  Use an exported function or constructor from the SDK`)
+    this.log(`  > const tx = new IronfishSDK.Transaction(Buffer.from('dsf3...', 'hex'))`)
+    this.log('')
 
     const historyPath = path.join(node.config.tempDir, 'repl_history.txt')
     await fs.mkdir(node.config.tempDir, { recursive: true })
     this.log(`Storing repl history at ${historyPath}`)
     this.log('Type .exit or press CTRL+C to quit')
 
     const server = repl.start('> ')
+    server.context.IronfishSDK = IronfishSDK
     server.context.sdk = this.sdk
```

Comment:
Why is this not accessible via `this.sdk`?

Reply:
`this.sdk` is an instance of the sdk not all the exported functions

Reply:
Why do the two have to be different? Why can't we access the exported functions from an instance?

Reply:
[This](https://github.com/iron-fish/ironfish/blob/master/ironfish/src/sdk.ts) is the SDK class. [This](https://www.npmjs.com/package/@ironfish/sdk) is the SDK library. Different things